logic/src/commonMain/kotlin/com/wire/kalium/logic/data/event/Event.kt

```diff
@@ -471,6 +472,19 @@ sealed class Event(open val id: String, open val transient: Boolean) {
             )
         }
 
+        data class MLSE2EIdUpdated(
+            override val id: String,
+            override val transient: Boolean,
+            val model: MLSE2EIdModel
```

Comment:
let's have the naming always without the d in Identity, I know the backend gives it like that, but we don't call it like that.
```suggestion
            val model: MLSE2EIModel
```